runtime/src/main/java/com/datastax/oss/driver/shaded/guava/common/primitives/UnsignedBytesSubstitutions.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.datastax.oss.driver.shaded.guava.common.primitives;
+
+import com.oracle.svm.core.annotate.Alias;
+import com.oracle.svm.core.annotate.Delete;
+import com.oracle.svm.core.annotate.RecomputeFieldValue;
+import com.oracle.svm.core.annotate.Substitute;
+import com.oracle.svm.core.annotate.TargetClass;
+import java.util.Comparator;
+
+@TargetClass(UnsignedBytes.LexicographicalComparatorHolder.UnsafeComparator.class)
+@Delete
+final class UnsafeComparatorSubstitutions {}
+
+@TargetClass(UnsignedBytes.LexicographicalComparatorHolder.class)
+final class LexicographicalComparatorHolderSubstitutions {
+
+  @Alias
+  @RecomputeFieldValue(kind = RecomputeFieldValue.Kind.FromAlias)
+  static Comparator<byte[]> BEST_COMPARATOR = UnsignedBytes.lexicographicalComparatorJavaImpl();
+
+  @Substitute
+  static Comparator<byte[]> getBestComparator() {
+    return UnsignedBytes.lexicographicalComparatorJavaImpl();
+  }
```

Comment:
There are two discrete ways to get an instance of an UnsafeComparator.  On 64-bit architectures it's the default stored in BEST_COMPARATOR, a value which is set by calling getBestComparator().  For sake of completeness we shut off both avenues of access to the now-deleted class; in fact we _have_ to do so.  If there's a viable code path which leads to a class we've explicitly marked as deleted Graal's analysis will detect it and (correctly) complain loudly.

Reply:
In the original code, the `getBestComparator()` method is invoked only by:
```
 static final Comparator<byte[]> BEST_COMPARATOR = getBestComparator();
```
So I think it should be enough to substitute `BEST_COMPARATOR` or `getBestComparator()` not both, right?

Reply:
I think you're right @tomekl007 .  I originally tried just replacing getBestComparator() before I noticed all the field references.  I knew I had to fix the field but I forgot to check after the fact to see if there were any other users of getBestComparator().

All that said, I still kinda like the idea of covering both of them.  It feels weird to me to only fix one and still leave the other open as a way to get at the (now-deleted) UnsafeComparator... knowing the whole thing only works because nobody's calling getBestComparator() right now.  I think I'd rather keep both in there.